lib/help.js

```diff
@@ -6,41 +6,41 @@ module.exports = `
 
     Options
         none             Display board view
-      --task, -t         Create task
-      --note, -n         Create note
-      --timeline, -i     Display timeline view
-      --delete, -d       Delete item
+      --archive, -a      Display archived items
       --check, -c        Check/uncheck task
-      --star, -s         Star/unstar item
+      --clear            Delete all checked items
       --copy, -y         Copy item description
-      --list, -l         List items by attributes
-      --find, -f         Search for items
+      --delete, -d       Delete item
       --edit, -e         Edit item description
+      --find, -f         Search for items
+      --help, -h         Display help message
+      --list, -l         List items by attributes
       --move, -m         Move item between boards
+      --note, -n         Create note
       --priority, -p     Update priority of task
-      --archive, -a      Display archived items
       --restore, -r      Restore items from archive
-      --clear            Delete all checked items
-      --help, -h         Display help message
+      --star, -s         Star/unstar item
+      --task, -t         Create task
+      --timeline, -i     Display timeline view
       --version, -v      Display installed version
 
     Examples
       $ tb
-      $ tb --task Make some buttercream
-      $ tb --task @coding Improve documentation
-      $ tb --task @coding @reviews Review PR #42
-      $ tb --note @coding Mergesort worse-case O(nlogn)
+      $ tb --archive
       $ tb --check 1 2
-      $ tb --delete 4
-      $ tb --star 2
+      $ tb --clear
       $ tb --copy 1 2 3
-      $ tb --priority @3 2
-      $ tb --timeline
+      $ tb --delete 4
       $ tb --edit @3 Merge PR #42
-      $ tb --move @1 cooking
       $ tb --find documentation
       $ tb --list pending coding
-      $ tb --archive
+      $ tb --move @1 cooking
+      $ tb --note @coding Mergesort worse-case O(nlogn)
+      $ tb --priority @3 2
       $ tb --restore 4
-      $ tb --clear
+      $ tb --star 2
+      $ tb --task @coding @reviews Review PR #42
+      $ tb --task @coding Improve documentation
+      $ tb --task Make some buttercream
+      $ tb --timeline
```

Comment:
Looks nice! The same should also be done on the [readme's Usage section](https://github.com/klaussinani/taskbook#usage) and the PR is good to be merged : )

Reply:
changes pushed